clearwater-auto-config/usr/share/clearwater-auto-config/bin/init-functions

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+# @file init-functions
+#
+# Copyright (C) Metaswitch Networks 2017
+# If license terms are provided to you in a COPYING file in the root directory
+# of the source code repository by which you are accessing this code, then
+# the license outlined in that COPYING file applies to your use.
+# Otherwise no rights are granted except for those provided to you by
+# Metaswitch Networks in a separate written agreement.
+
+# This file provides basic shared init functionality between the different
+# auto-config varieties
+
+# Include basic functions to add and remove sections from files
+. /usr/share/clearwater-auto-config/bin/common
```

Comment:
It's copied from clearwater-infrastructure/usr/share/clearwater/infrastructure/install/common by the install script.